kde-apps/kio-extras/kio-extras-9999.ebuild

```diff
@@ -60,11 +56,15 @@ DEPEND="
 	openexr? ( media-libs/openexr:= )
 	phonon? ( >=media-libs/phonon-4.12.0[qt6(+)] )
 	samba? (
+		>=kde-frameworks/kdnssd-${KFMIN}:6
 		net-fs/samba[client]
 		>=net-libs/kdsoap-2.2.0:=[qt6(+)]
 		>=net-libs/kdsoap-ws-discovery-client-0.3.0
 	)
-	sftp? ( net-libs/libssh:=[sftp] )
+	sftp? (
+		dev-libs/qcoro
```

Comment:
this should actually be a `DEPEND`-only, can you check?

Reply:
we don't generally keep these in RDEPEND if stripped by --as-needed - cmake `target_link_libraries` is not authoritative methinks @thesamesam

Reply:
My rationale is that this is an implementation detail so it could change later without notice (especially if upstream documentation instructs to link it). So I wouldn't want to look at it again for the off chance if gets changed in qcoro.

Reply:
We should include it if it's documented especially, because it could be that some e.g. new template gets instantiated and suddenly it is needed.

Reply:
I've basically not `RDEPEND`ed on `dev-libs/qcoro` anywhere yet. Could you create a separate PR to deal with all affected revdeps?